usagov/usagov-2021
# USAGOV-1696 - Fix error output when no arguments are provided

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1696

## Description


Add argument checking and appropriate error output

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

In your local development environment, run: `bin/deploy/git-annotation-parser.sh`
The output should be: `First argument must be a valid CF space name`

Then run `git fetch --all --tags`  (this will make sure you have access to all the git tags which this command needs)

Now run: `bin/deploy/git-annotation-parser.sh prod`
The output should be something like:
```
To deploy the waf, please execute the following command:

   ROUTE_SERVICE_APP_NAME=waf \
       ROUTE_SERVICE_NAME=waf-route-prod-usagov \
       PROTECTED_APP_NAME=cms \
          bin/cloudgov/deploy-waf 12018 @sha256:bc7ac56f9e00827aaa45fe62ba77463eda93d4fb6769d5ed14992fb582

To deploy the cms, please execute the following command:
   bin/cloudgov/deploy-cms 12018 @sha256:131f9a4f7f5925d8c676adb2389847bbf18ed37608d764f5de83a4c1fe999c81

To deploy the static site, please execute the following command:
   bin/cloudgov/deploy-www 12018 @sha256:49412a205b8471fc6019c4a2cb8e4d60dde8b8366922aa43ffe8db9ea20272cb
```

The reviewer should confirm functionality on MacOS


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
